crates/l2/Makefile

```diff
@@ -170,7 +176,11 @@ restart-l2: down-l2 rm-db-l2 init-l2 ## 🔄 Restarts the L2 Lambda ethrex Clien
 
 init-prover: ../../target/release/ethrex_prover ## 🚀 Initializes the Prover
 	CONFIGS_PATH=${ethrex_L2_CONFIGS_PATH} \
-	../../target/release/ethrex_prover
+	../../target/release/ethrex_prover -- \
+	--http.addr http://localhost \
+  	--http.port 3900 \
+	--prover-server-endpoint \
```

Comment:
Think this is no longer needed

Reply:
Good catch.

Reply:
Done in [a460a51](https://github.com/lambdaclass/ethrex/pull/2744/commits/a460a516e8fda2c9a9e2bcc305af2fab686fc089)